website/docs/guides/1.15.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1150-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.15.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+
+
+**Breaking Changes:**
+
+Format of `okta_idp_id` that contains the IdP Id in `federated_settings_identity_provider` resource now accepts a different format to align with Atlas API. Both the current and new IdP Id format are accepted in terraform-provider 1.15.0 version.
```

Comment:
Yes you are right. 
Right now we will accept both Ids and when using the old one return a warning. As you said, OIDC features will only be available to users using new Id. 
 I put it under the `Breaking Changes` section even though it's not a breaking changes (users will only get a warning) to increase visibility of this to the users. Now that you mention it it will be confusing for users, I will move it to `Deprecations and Removals:  ` section. Thanks for the feedback.